MIP/mip-18/README.md

```diff
@@ -0,0 +1,96 @@
+   - Role: Proposing upgrades.
+   - Addresses: Deployed on Ethereum, Sepolia, and Holesky (if available).
+   - Signers: Andy Bell (0x49F86Aee2C2187870ece0e64570D0048EaF4C751), Richard Melkonian (0xaFf3deeb13bD2B480751189808C16e9809EeBcce), Liam Monninger (0x12Cbb2C9F072E955b6B95ad46213aAa984A4434D), Primata (0xB2105464215716e1445367BEA5668F581eF7d063) (Engineers involved with Token), and a remote key for automated transaction proposals via SafeKit API's [proposeTransaction](https://docs.safe.global/sdk/api-kit#propose-a-transaction-to-the-service).
+   - Justification: We leave some wiggle room incase one of the engineers is away. We still need 3/4 engineers to sign.
+
+2. **Movement Foundation Multisig** (3/5 signatures required, triggerable after 2-day timelock):  
+   - Role: Executing upgrades.
+   - Addresses: Deployed on Ethereum, Sepolia, and Holesky (if available).
+   - Signers: TBD.
+
+Both multisigs are part of a timelock mechanism to enforce a 2-day waiting period between the proposal and execution of upgrades. The engineers' multisig schedules the upgrade, while the foundation multisig executes it after the timelock expires.
+
+### Workflow
+
+![Workflow](workflow.png)
+
+1. **Deployment**:  
+   - The Contract Pipeline deploys a new contract implementation and generates transaction data for the upgrade.
+
+2. **Proposal**:  
+   - The remote key automates the submission of a transaction proposal using SafeKit API’s `proposeTransaction`, populating the transaction for review by Core Contributors engineers.
+   - The proposal includes the transaction for `timelock.schedule(proxyAdmin.upgradeAndCall(newContractImplementation))`.
+
+3. **Approval**:  
+   - The Core Contributors engineers review and gather 4/5 signatures to approve the proposal.
+   - Once approved, the `timelock.schedule` transaction is enacted, holding the upgrade in a 2-day timelock.
+
+4. **Execution**:  
+   - After the timelock period, the Movement Foundation multisig gathers 3/5 signatures to execute the upgrade.
+   - This triggers the `timelock.execute` transaction, allowing the upgrade to be applied to the ProxyAdmin, which will perform the contract upgrade.
+
+### Multisig Security Considerations
+
+- Believe it is best practice to rely on a service that is fully dedicated to multisigs. Eventual exploits found might open us up to exploits while safe.global could've prevent the exploit for us.
+- Having full control of our multisig deployments is dangerous and more engineering debt.
+- Having manual verification might stop us from automating a exploit that we did not notice.
+- Both multisigs serve as a dual signing authorities, ensuring a strict separation between proposing and executing transactions.
+- The timelock ensures a sufficient review window for all parties, with the ability for either multisig to cancel the transaction during the delay period.
+- Multisig can fully transition to a fully automated KMS setup once we have that in place.
```

Comment:
I would provide a note indicating the original deployment date and that this was indeed not initially implemented with KMS.